core/include/optee_rpc_cmd.h

```diff
@@ -107,8 +107,7 @@
  */
 #define OPTEE_RPC_CMD_SHM_FREE		7
 
-/* Was OPTEE_RPC_CMD_SQL_FS, which isn't supported any longer */
-#define OPTEE_RPC_CMD_SQL_FS_RESERVED	8
+#define OPTEE_MSG_RPC_CMD_I2C_TRANSFER	8
```

Comment:
Please don't reuse this id, allocate a new.

Reply:
ok